region/maridia/outer/Fish Tank.json

```diff
@@ -489,6 +490,30 @@
         "h_canCrystalFlash"
       ]
     },
+    {
+      "link": [2, 4],
+      "name": "Grapple Teleport into Grapple Jump",
+      "entranceCondition": {
+        "comeInWithGrappleTeleport": {
+          "blockPositions": [[5, 3]]
+        }
+      },
+      "requires": [
+        {"doorUnlockedAtNode": 4},
+        "canGrappleJump",
+        "canInsaneJump"
```

Comment:
- You can't avoid the fish by dead hanging, but you can learn a simple movement pattern to avoid it
- I didn't think of turning off grapple while leaving the beam out